src/Http/Http.Abstractions/src/Extensions/UseExtensions.cs

```diff
@@ -14,6 +14,15 @@ public static class UseExtensions
     {
         /// <summary>
         /// Adds a middleware delegate defined in-line to the application's request pipeline.
+        /// <para>
+        /// Prefer using <see cref="Use(IApplicationBuilder, Func{HttpContext, RequestDelegate, Task})"/> for better performance as shown below:
+        /// <code>
+        /// app.Use((context, next) =>
+        /// {
+        ///     return next(context);
+        /// });
+        /// </code>
```

Comment:
Could you also add a quip about preferring `Run` if `next` is  unused?

Reply:
Done